### DISCUSSION TITLE
How do I configure `lualine.nvim` to follow catppuccin theme

### DISCUSSION THREAD

**Discussion by omar-zahid** [catppuccin/nvim#677] [Q&A] [answered]
I've followed the guide, and added `theme = "catppuccin"` to options but still not getting the correct colors.

![image](https://github.com/catppuccin/nvim/assets/57479337/d0db5662-670d-479d-bb0e-b48f04caa79f)

I'm expecting the following colors:

![image](https://github.com/catppuccin/nvim/assets/57479337/8f611406-0d3c-49f0-9755-9d2e80295983)

Appreciate it. Thanks.

**Comment by iamvimuserbtw** [comment, marked as answer]
Hi there. This has been said before but the screenshot you show with the expecting colors and look, uses feline and not lualine.
